spec/sample-data/valid/control-statement-04.json

```diff
@@ -0,0 +1,18 @@
+{
+  "subject_entity": {
+    "entity_properties": {
+      "company_number": "09876543",
+      "jurisdiction_code": "gb"
+    },
+    "entity_type": "company"
+  },
+  "confidence": "HIGH",
+  "control_level": "unknown",
+  "no_owners_reasons": [
+    "Super Secure Person With Significant Control: The person with significant control's details are not shown because restrictions on using or disclosing any of the individual’s particulars are in force under regulations under section 790ZG in relation to this company"
+  ],
+  "confidence": "HIGH",
+  "retrieved_at": "2015-08-11T23:00:08+01:00",
+  "sample_date": "2016-01-01",
+  "source_url": "http://download.companieshouse.gov.uk/en_pscdata.html"
+}
```

Comment:
If the only difference between this and `03` is the `no_owners_reasons` then I suggest we drop it. The value of that field is irrelevant from the schema point of view so there's no value to having two examples of it.